src/contexts/globalLayout/GlobalNavbar/CollectionSaveButtonWithCaption.tsx

```diff
@@ -101,63 +92,35 @@ export function CollectionSaveButtonWithCaption({
     <StyledConfirmationContainer>
       <StyledButton
         onClick={isPopupDisplayed ? handleCloseCaption : handleOpenCaption}
-        disabled={disabled}
+        disabled={disabled || !hasUnsavedChange}
         pending={isLoading && !isWhiteRhinoEnabled}
       >
         <HStack gap={4} align="center">
           {label}
-          {isWhiteRhinoEnabled && (
-            <StyledChevronSvg
-              isActive={isPopupDisplayed}
-              width="12"
-              height="7"
-              viewBox="0 0 12 7"
-              fill="none"
-              xmlns="http://www.w3.org/2000/svg"
-            >
-              <path
-                d="M1.33337 1L6.00004 5.66667L10.6667 1"
-                stroke="#FEFEFE"
-                strokeMiterlimit="10"
-              />
-            </StyledChevronSvg>
-          )}
         </HStack>
       </StyledButton>
+
+      {/* Used to hijack click events on things outside of the caption area */}
+      {isPopupDisplayed && <Backdrop onClick={handleCloseCaption} />}
+
       <StyledCardContainer gap={hasUnsavedChange ? 12 : 24} isActive={isPopupDisplayed}>
-        {isActive && (
-          <>
-            <HStack justify="flex-end">
-              <StyledCloseButton onClick={handleCloseCaption}>
-                <CloseIcon />
-              </StyledCloseButton>
-            </HStack>
-            {hasUnsavedChange ? (
-              <>
-                <StyledConfirmationContent gap={8}>
-                  <BaseM>Share your update to the feed with an optional note.</BaseM>
-                  <TextAreaWithCharCount
-                    currentCharCount={caption.length}
-                    maxCharCount={600}
-                    onChange={handleCaptionChange}
-                    hasPadding
-                    defaultValue={caption}
-                    placeholder="Add a note..."
-                    textAreaHeight="50px"
-                  />
-                </StyledConfirmationContent>
-
-                <Button onClick={handleSubmit} disabled={isSaveDisabled} pending={isLoading}>
-                  Save
-                </Button>
-
-                {error && <ErrorText message={error} />}
-              </>
-            ) : (
-              <StyledNoChangesTitle>No changes made yet.</StyledNoChangesTitle>
-            )}
-          </>
-        )}
+        <VStack gap={12}>
+          <TextAreaWithCharCount
```

Comment:
are the dimensions correct? in the figma the box looks a little shorter! (figma screenshot below)

![image](https://user-images.githubusercontent.com/12162433/202376558-e5687e44-e2f9-4e83-ba07-dd936f221fe1.png)

Reply:
just tried decreasing the height and it looks pretty tight. I think the taller version is a bit better user experience

<img width="268" alt="Screen Shot 2022-11-17 at 20 23 09" src="https://user-images.githubusercontent.com/80802871/202433662-f2290284-e3e4-4ac6-a92b-105ad954f3cf.png">